python/django/python-hello-world/kubernetes-manifests/hello.deployment.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: python-hello-world-app
+  labels:
+    app: python-hello-world
+    tier: app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: python-hello-world
+      tier: app
+  template:
+    metadata:
+      labels:
+        app: python-hello-world
+        tier: app
+    spec:
+      initContainers:
+      # Wait for the database to get ready before starting the server, otherwise
+      # the server will crash
+      - name: check-db-ready
+        image: postgres
+        command: ['sh', '-c',
+          'until pg_isready -h python-hello-world-db.default.svc.cluster.local -p 5432;
```

Comment:
why do you need this for hello-world?

Reply:
Removed database for hello-world template.